Framework/jmaqs-webservices/src/test/java/com/magenic/jmaqs/webservices/MediaTypeTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 (C) Magenic, All rights Reserved
+ */
+
+package com.magenic.jmaqs.webservices;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
```

Comment:
Could this be removed? IntelliJ says it isn't in use

Reply:
Will do.  IntelliJ has been adding that as part of the template for some reason.